integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/RestServiceSpec.scala

```diff
@@ -142,18 +143,64 @@ class RestServiceSpec extends TestSpecBase {
 
   "cluster information" should {
     "retrieve 1 master for a non-HA cluster" in {
+      // setup
+      val expectMastersCount = cluster.getMasterHosts.size
+      val expectMaster = cluster.getMasters.head
 
-    }
-
-    "retrieve 0 worker, if cluster is started without any workers" in {
-      // todo: defect
+      // exercise
+      restClient.listMasters().size shouldEqual expectMastersCount
```

Comment:
Suggestion: store the result of `listMasters()` as a variable. Do not call the methods at every verification step.